### ISSUE TITLE
Hardened malloc / Attempt to play video from camera crashes app

### ISSUE THREAD

**Issue by ctrl-zed** [deckerst/aves#764]
**Describe the bug**
After a fairly recent update (past couple months I'm guessing), I am no longer able to play videos I have taken with my phone. I can see the videos listed in the Camera folder, but when I tap the thumbnail the app crashes.

**To Reproduce**
1. Go to Pinned Camera folder 
2. Click on any video file 

**Expected behavior**
Expected the video to play.

**System information and logs:**
Package: deckers.thibault.aves.libre
Installer: dev.imranr.obtainium
Aves version: 1.9.5-libre, build 10602
Flutter: stable 3.13.4
Android version: 13, API 33
Android build: TQ3A.230805.001.2023090600
Device: Google Pixel 7
Geocoder: not available
Mobile services: not available
System locales: en_CA
Storage volumes: /storage/emulated/0/
Storage grants: /storage/emulated/0/Pictures/, /storage/emulated/0/DCIM/
Error reporting: false

**Additional context**
I have been using the stock galley app to watch videos since the bug began. I am only just now reporting the issue.

**Comment by ctrl-zed** [comment]
Sorry I have never used GitHub, I'm not familiar with how to upload a log file. Here is a link via wormhole: 

https://wormhole.app/Kz9q0#HsyWyHAjdh6WSHYn3gzy8g

**Comment by ctrl-zed** [comment]
I'm sorry, didn't mean to close 😑

**Comment by Biscorn** [comment]
A commit has been made about it and should have fix it 4 hours ago. But still, the problem is not gone. We will wait for the next patches.

**Comment by deckerst** [comment]
Thanks for reporting, and welcome to GitHub!

Do you use GrapheneOS? if so, this is likely a duplicate of this issue:
- #722 

But i can't do anything myself to fix it, it will depend on the goodwill of the maintainer of the video library I'm using.

@Biscorn the recent commit is for a different issue that was happening when cataloguing some specific videos. It wasn't related to playback.

**Comment by Biscorn** [comment]
> Thanks for reporting, and welcome to GitHub!
> 
> Do you use GrapheneOS? if so, this is likely a duplicate of this issue:
> - #722 
> 
> But i can't do anything myself to fix it, it will depend on the goodwill of the maintainer of the video library I'm using.
> 
> @Biscorn the recent commit is for a different issue that was happening when cataloguing some specific videos. It wasn't related to playback.

Thanks for your reply ! Same for me, I am using GrapheneOS and though your commit was for this issue we related.

**Comment by ctrl-zed** [comment]
@deckerst Yes you're right, I looked through issue #722 and it is the same problem I'm having. I am using GrapheneOS. I toggled the Exploit Protection Compatibility Mode option and it has solved the issue.

I'll close this issue as it is a duplicate. I appreciate your quick reply. I will follow the progress on issue #722.

Thank you also for such a great app. I've been using it for months now and very much enjoy it.
